manifests/init.pp

```diff
@@ -34,6 +34,7 @@
           fail("nfs module only supports lsbdistid Debian and Ubuntu of osfamily Debian. Detected lsbdistid is <${::lsbdistid}>.")
         }
       }
+      $default_nfs_service_required_svcs = undef
```

Comment:
Added for completeness, i.e. so this new variable would have a value assigned (even if undef) for each clause of case statement.